test/system/post_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index/show', type: :system do
+  describe 'post index page' do
```

Comment:
The Post index/show tests are missing the following:
- a test for the user image
- the no. of posts made
- how many comments are on the page
- how many likes are on the page

Reply:
- [ ] As requested by the last reviewer, Please add those tests to the Post index/show views:

- a test for the user image
- the no. of posts made
- how many comments are on the page
- how many likes are on the page